src/Aspire.Hosting.Azure.SignalR/Aspire.Hosting.Azure.SignalR.csproj

```diff
@@ -11,11 +11,13 @@
   <PropertyGroup>
     <MinCodeCoverage>96</MinCodeCoverage>
   </PropertyGroup>
-
+  
   <ItemGroup>
     <ProjectReference Include="..\Aspire.Hosting.Azure\Aspire.Hosting.Azure.csproj" />
     <PackageReference Include="Azure.Provisioning" />
     <PackageReference Include="Azure.Provisioning.SignalR" />
+    <PackageReference Include="Microsoft.Azure.SignalR.Management" />
```

Comment:
I think the convention for other services is to use the Service SDK to init a client for healthcheck. In my opinion, it looks more clean that way as well. Also the tests use this SDK to get negotiate URL as well

Reply:
Microsoft.Azure.SignalR.Management is a little bit different story, and it references Microsoft.Azure.SignalR which references aspnetcore SignalR. It sounds too much for a project doing the service provisioning

Reply:
Okay makes sense, I'll just try using REST API for the health check in the main code, and maybe use this Management SDK to perform the tests since it's more complicated to test with using just REST API